pyiceberg/table/__init__.py

```diff
@@ -224,6 +225,9 @@ class TableProperties:
     MANIFEST_MERGE_ENABLED = "commit.manifest-merge.enabled"
     MANIFEST_MERGE_ENABLED_DEFAULT = False
 
+    METADATA_PREVIOUS_VERSIONS_MAX = "write.metadata.previous-versions-max"
+    METADATA_PREVIOUS_VERSIONS_MAX_DEFAULT = 100
```

Comment:
It will be even better if we could update the doc to include this new option:
https://github.com/apache/iceberg-python/blob/846713b18c4f0adbb75fda14d9b863464e91d4bc/mkdocs/docs/configuration.md?plain=1#L29-L38

Reply:
Sure, I had this in mind when I started to work this but then forgot about it. Will the property in the doc.

Reply:
@HonahX I've updated the docs.